gatsby-node.js

```diff
@@ -20,12 +20,18 @@ exports.onPostBuild = async function(
     query,
     transformer = identity,
   }) {
+    if (!query) {
+      report.panic(`failed to index to Algolia. You did not give "query" to this query`)
```

Comment:
Since it's an error that is specific to a certain parameter presence, maybe you could explain what query is for?

Reply:
It's clear enough like this I think, it's the only required parameter so it's the only thing documented :)